assets/src/ba_data/python/bastd/game/meteorshower.py

```diff
@@ -164,7 +164,7 @@ def handlemessage(self, msg: Any) -> Any:
             # Augment standard behavior.
             super().handlemessage(msg)
 
-            death_time = ba.time()
+            death_time = ba.time(timeformat=ba.TimeFormat.MILLISECONDS)
```

Comment:
Thanks for fixing any breakage here, but I may go through and try to convert this to seconds. I'd love to standardize on using float seconds whenever possible just to avoid ugly ba.TimeFormat.MILLISECONDS args everywhere.